### DISCUSSION TITLE
Border around grouped nodes

### DISCUSSION THREAD

**Discussion by tmarsik42** [open-source-labs/Svelvet#244] [Q&A] [answered]
Can i make a border or otherwise visualize grouped nodes? I need to do something like on image.

<img width="455" alt="Snímek obrazovky 2023-03-13 v 12 19 15" src="https://user-images.githubusercontent.com/95856146/224687215-03c9fc76-8d14-4ee4-b012-fbe8e44b1250.png">

**Comment by briangregoryholmes** [comment, marked as answer]
This is a feature in Svelvet 7! Expect more functionality around this soon!

https://svelvet.mintlify.app/components/group
